### PR TITLE
PR for: Docs: Installation instructions

### PR DESCRIPTION
**Pull request by joeflack4**
Updates
```
Added installation instructions, including additional steps for `SqlImplementation`
```

Addresses: #233

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -223,6 +223,18 @@ OAK uses [`pystow`](https://github.com/cthoyt/pystow) for caching. By default,
...
 
+## Installation
+### General purpose installation
+1. Python 3.9+
+2. Run: `pip install oaklib`
+
+### SqlImplementation requirements
+If using `SqlImplementation`, some additional steps beyond _general purpose installation_ are required.
+1. Install [`sqlite`](https://www.sqlite.org/download.html)
+2. Install [`relation-graph`](https://github.com/balhoff/relation-graph#installation)
```
Comment (joeflack4) [inline README.md:234]: Depends on: https://github.com/balhoff/relation-graph/pull/146

File: README.md [hunk 2]
```diff
@@ -223,6 +223,18 @@ OAK uses [`pystow`](https://github.com/cthoyt/pystow) for caching. By default,
...
+4. Install [`rdftab`](https://github.com/ontodev/rdftab.rs#installation)
```
Comment (joeflack4) [inline README.md:235]: Depends on: https://github.com/ontodev/rdftab.rs/pull/22

**Review by cmungall — CHANGES_REQUESTED** [review]
Thanks for the contribution! We can include the base installation instructions

For the sqlite part: Let's not duplicate this documentation everywhere

remember, you don't need relation-graph or even sqlite to use SqlImplementation.

the complicated install of relation-graph and rdftab is only required for compiling a sqlite from owl source.

99% of OAK users won't need this, they will be using a pre-generated sqlites e.g. via `sqlite:obo:mondo`, and for this an pure python install from PyPI is sufficient.

and even for the smaller group who need to do the compliation, the complicated steps will disappear soon: https://github.com/INCATools/semantic-sql/issues/41

the appropriate place for the granular description is in this tutorial - can you give this a read over and make any PRs there?

https://incatools.github.io/ontology-access-kit/intro/tutorial07.html

**Comment by joeflack4** [comment]
@cmungall Thanks for looking this over.

Done on the new PR: https://github.com/INCATools/ontology-access-kit/pull/242

I must say, I don't enjoy `.rst` outside of the usefulness of toc tree.

Also, just made some edits to this PR to remove the `SqlImplementation` part of the installation, if you want to re-open and merge this one. I know it's pretty obvious, but I like to leave things simple for Jr devs who may not have Python experience.

**Pull request closed without merging**